### PR TITLE
Fix regex used to split course name

### LINKED ISSUE #197: Full Course ID not being displayed
For courses like CS/ECE/EEE/INSTR F211 Microcroproc & Interfacing, the full course ID is not shown. Only INSTR F211 is shown. This might be a problem, especially if branch wise courses are made on Moodle (as was the case in Sem 1, 2019-2020).

### PR DESCRIPTION
**Pull request by VaishnaviShri**
close #197

### CODE REVIEW

**Comment by VaishnaviShri** [comment]
The course names with codes separated with '/' like "CS/ECE/EEE/INSTR F241" are now displayed completely now.


WhatsApp Image 2020-05-17 at 10 31 56 AM

File: app/src/main/java/crux/bphc/cms/models/Course.java [hunk 1]
```diff
@@ -114,7 +114,7 @@ public String getShortname() {
 
     public String[] getCourseName(){
         String courseName = getShortname();
-        String regex =  "([\\w\\d \\-'&,]+ \\w\\d\\d\\d) ([\\w\\d \\-'&,]+) ([LTP]\\d*)";  // Specifies the string pattern which is to be searched
+        String regex =  "([\\w\\d \\-/'&,]+ \\w\\d\\d\\d) ([\\w\\d \\-/():+\"'&.,?\\[\\]]+) ([LTP]\\d*)";  // Specifies the string pattern which is to be searched
```
Comment (abhijeetviswa) [inline app/src/main/java/crux/bphc/cms/models/Course.java:117]: Any specific reason as to why `/():+\"'&.,?\\[\\]` was also included? Are characters like paranthesis and `?` seen in course names?

Comment (VaishnaviShri) [reply app/src/main/java/crux/bphc/cms/models/Course.java:117]: Parenthesis have been used in certain course names like "CE F243 SOIL MECHANICS T1 (SAT)".
Adding other characters was probably unnecessary. I'll change that.

Comment (abhijeetviswa) [reply app/src/main/java/crux/bphc/cms/models/Course.java:117]: Remove the square brackets. I don't think those will be used.

Comment (abhijeetviswa) [reply app/src/main/java/crux/bphc/cms/models/Course.java:117]: Everything can be left behind.

**Comment by abhijeetviswa** [comment]
Everything looks good.

**Pull request merged**